Makefile

```diff
@@ -0,0 +1,50 @@
+DOCKER_NAME = go-module
+DOCKER_HOSTNAME = ghcr.io
+DOCKER_NAMESPACE = aradhalevy
```

Comment:
DOCKER_NAMESPACE ?= fybrik

Reply:
While you are right that it should be Fybrik, for now until this PR is approved the image is locally at my github (aradhalevy), and after it will be approved I will push the images to this repo and change every "aradhalevy" to "fybrik"